src/components/AppLayout/index.tsx

```diff
@@ -92,6 +93,7 @@ const Layout: React.FC<Props> = ({
 }): React.ReactElement => {
   const [mobileNotSupportedClosed, setMobileNotSupportedClosed] = useState(false)
   const { pathname } = useLocation()
+  const { onClickInstall } = useMobileInstallHandler()
```

Comment:
Could you call this inside the `<MobileNotSupported>` component and handle all of the conditional logic within that component?